force-app/main/default/lwc/crmMessagingQuickText/crmMessagingQuickText.js

```diff
@@ -254,6 +264,28 @@ export default class CrmQuickText extends LightningElement {
         }
     }
 
+    handleCheckBoxChange(event) {
+        let signature = this.template.querySelector('.standardSignature');
+        let noteArea = this.template.querySelector('.conversationNoteTextArea');
+        if (event.detail.value.includes('Standard')) {
+            this.checkBoxValue = 'Standard';
+            signature.className = signature.className
+                .split(' ')
+                .filter((e) => e !== 'signature-textarea-hidden')
+                .concat('signature-textarea')
+                .join(' ');
```

Comment:
For å legge til og slette klasser i js kan du heller bruke classList og remove() og add()

Reply:
+1